angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -48,21 +89,45 @@ export const defaultStory = () => ({
         sprk-c-Card__content
         sprk-o-Stack
         sprk-o-Stack--medium">
-        Base Card Content
+        <p class="sprk-TypeBodyTwo">Default Card</p>
       </div>
     </sprk-card>
   `,
 });
 
-defaultStory.story = {
-  name: 'Default',
+legacyStory.story = {
+  name: 'Legacy (Deprecated)',
   parameters: {
-    docs: { iframeHeight: 150 },
```

Comment:
Did this get removed intentionally?

Reply:
yeah, the default was 200 and it looked better so i removed the 150